src/js/story-factory.js

```diff
@@ -1,31 +1,48 @@
-const Story = (title) => {
+/* Removed findFirstChapter. Added chapters sorting in addChapters method. I added getCurrentChapter which is more felxible because find method
+always returns first element that meets given requirements so in every new story this element is first chapter (thanks to sort method) and it 
+also finds first chapter which completion status is false.
+
+I added quiz parameter since i realized how and when quizes should be displayed, then it makes sense to pass all story quizzes when we initialize
+new story object. Also, like i wrote in game logic - getFinalQuizzes probably shold be changed to method similar to getCurrentChapter.
+ */
```

Comment:
Alright yeah changes of how to get chapters make sense to me. (Haven't checked it in detail though)

`getFinalquizzes` for sure needs to be somehow changed. @emdevs wrote the QuizSummary factory in her last PR which was merged recently. I think you can make use of that to get the all the quizzes of the story and if needed add there a method to get the current quiz.